app/views/catalog/_facet_pagination.html.erb

```diff
@@ -1,3 +1,13 @@
+    <% if @facet.index_pagination && params["facet.sort"] == 'index' %>
+      <div class="alpha-filter">
+        <%= t('blacklight.search.facets.filter') %>:
+        <%= link_to(t('blacklight.search.facets.clear'), @pagination.params_for_resort_url('index', params.except(:"facet.prefix")), :data => {:ajax_modal => "preserve"}) %>
+        <% ('A' .. 'Z').to_a.each do |letter| %>
```

Comment:
I think this range should be part of the facet configuration. It seems like it's both language and data dependent.

Also, do you know if `facet.prefix` is case-sensitive?